### DISCUSSION TITLE
.github repository and default templates and issues

### DISCUSSION THREAD

**Discussion by lucasrabaca** [community/community#38170] [Repositories] [answered]
Having set the .github repository to public, my other private projects don't get the default files correctly, only the other public repositories.
am i doing this right or there is no way to set default files for private repositories in my organization?

In the docs only the .github repository is mentioned to be public.
[Docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file).

**Comment by david-engelmann** [comment]
[From this documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file), It states "Default files will be used for any repository owned by the account that does not contain its own file of that type." Do your private repos have .github folders for workflows or what not?

**Reply by lucasrabaca** [reply]
Yes, the repository have:

.github/workflows/...
docs/

both folders doesn't have pull_request_template.md or ISSUE_TEMPLATE folder.

In this specific repository, the pull_request_template from .github repository is working as expected, but the issue is still blank without the option to select an template.


In the .github repo and in other public repositories i can select the issue template normally.

**Reply by lucasrabaca** [reply]
I have another repository without the docs and .github folder, issue templates doesn't work, only the pull_request_template.

**Comment by lucasrabaca** [comment, marked as answer]
I've got it working, turns out i've duplicated the config.yml to create the templates and ended up with the **.yml** extension...
renamed **bug_report.yml** to **bug_report.md**.

It is really strange for them to keep working on the .config repository, even with wrong extension...
